core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1274,6 +1274,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_METRICS_TIME_SERIES_INTERVAL =
+      new Builder(Name.MASTER_METRICS_TIME_SERIES_INTERVAL)
+          .setDefaultValue("5min")
+          .setDescription("Interval for which the master records metrics information. This affects "
+              + "the granularity of the metrics graphed in the UI.")
```

Comment:
Is this equal to the time interval charted in the UI?

Reply:
I think we might do something fancy in the UI, where if this because a very large payload (ie. 1 month of data about 8.5k entries), we will display less of the older data.